### ISSUE TITLE
doesn't work

### ISSUE THREAD

**Issue by samyycX** [Kxnrl/sm-ext-ArmsFix#3]
i have installed this extension on my server and create a ArmsFix.autoload file. 
It displayed in "sm exts list". but my glove still overlap on my custom model.
im using Windows

**Comment by Kxnrl** [comment]
uninstall all gloves plugin

**Comment by samyycX** [comment]
> uninstall all gloves plugin

I tried, not work

**Comment by Kxnrl** [comment]
can you share screenshot here?

**Comment by samyycX** [comment]
> can you share screenshot here?

plugin list or extension list?

**Comment by Kxnrl** [comment]
overcap arms

**Comment by samyycX** [comment]
> overcap arms

![ab](https://user-images.githubusercontent.com/60744529/115387346-1ffed100-a20d-11eb-8c24-6ae99cf5110b.png)

**Comment by samyycX** [comment]
> overcap arms

I just tried to uninstall all plugins, not working too

**Comment by crashzk** [comment]
For me, too, everything was working fine, now it started to bug the arms of the custom skins again.

I already removed gloves, ws, knife, agents, everything, I left only the custom skins and nothing, the bug continues.

Any corrections that we can apply?

**Comment by crashzk** [comment]
I tested it with this plugin: https://forums.alliedmods.net/showthread.php?t=293295

Nothing works either.

**[EDIT1]**
From what I saw too, if I don't have any gloves and use a glove plugin and don't choose a glove, the extension removes your hand, I removed the extension and the problem solved it.

Server console:
```
Late precache of models/weapons/v_models/arms/glove_hardknuckle/v_glove_hardknuckle.mdl

Late precache of models/weapons/v_models/arms/glove_hardknuckle/v_glove_hardknuckle.mdl

Attempted to create unknown entity type cs_bot_patrol_route_waypoint!

Can't init cs_bot_patrol_route_waypoint

Attempted to create unknown entity type cs_bot_patrol_route_waypoint!

Can't init cs_bot_patrol_route_waypoint

Attempted to create unknown entity type cs_bot_patrol_route_waypoint!

Can't init cs_bot_patrol_route_waypoint

Attempted to create unknown entity type cs_bot_patrol_route_waypoint!

Can't init cs_bot_patrol_route_waypoint

Late precache of models/weapons/v_models/arms/glove_hardknuckle/v_glove_hardknuckle.mdl
```

**Comment by crashzk** [comment]
Can we expect any fixes for this extension @Kxnrl?

**Comment by Kxnrl** [comment]
> Can we expect any fixes for this extension @Kxnrl?

No, this extension made for custom player skin. If not use custom player skin and arms, this extension will make your arms invisible

**Comment by crashzk** [comment]
> > Can we expect any fixes for this extension @Kxnrl?
> 
> No, this extension made for custom player skin. If not use custom player skin and arms, this extension will make your arms invisible

Ok, but even the standard CSGO gloves? The ones you can buy on the Steam Market.

I also tested with custom skins with arms, but even so it didn't show the skins' arms.
